src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java

```diff
@@ -28,12 +26,21 @@
     private Task verifyCapstoneTask;
     private final Logger logger;
     private final File destinationDir;
+    //this is for testing
+    private final Function<Project, List<RemoteRepository>> getRepositoriesFunction;
```

Comment:
This allows us to inject a test-controllable object to get repositories.  Before, it was a static call so unable to test.